settings.json

```diff
@@ -4,7 +4,7 @@
     "MAIN_GUILD": 781828787352240129,
     "MUSIC_TEXT_CHANNEL": 1216685839585771560,
     "MUSIC_VOICE_CHANNEL": 1216674440486588457,
-    "ALLOWED_CATEGORY_IDS": [987352501172989993, 1144810748158165043],
+    "ALLOWED_CATEGORY_IDS": [987352501172989993, 1144810748158165043, 1143358482494533662],
```

Comment:
which channel this ID is linked to? >.<

Reply:
It's my test server category. Added it for testing in that. will remove it